st2client/st2client/formatters/table.py

```diff
@@ -143,11 +146,16 @@ def format(cls, entries, *args, **kwargs):
                 else:
                     value = cls._get_simple_field_value(entry, field_name)
                     transform_function = attribute_transform_functions.get(field_name,
-                                                                          lambda value: value)
+                                                                           lambda value: value)
                     value = transform_function(value=value)
                     value = strutil.unescape(value)
                     values.append(value)
             table.add_row(values)
+
+        # width for the note
+        global table_width
```

Comment:
I will test it and go ahead and merge it if everything look OK so it will end up in v2.3.1, but we really need to refactor and fix this global variable thing since it's not robust and could cause issues.